test/import/FSHImporter.test.ts

```diff
@@ -466,6 +466,56 @@ Long statement:
     expect(profile.description).toBe(expectedDescriptionLines.join('\n'));
   });
 
+  it('should parse numbers using exponential notation', () => {
+    const input = leftAlign(`
+    Instance: MyObservation
+    InstanceOf: Observation
+    * component[0].valueQuantity = 2.3E11 'kg'
+    * component[1].valueQuantity = 155e-8 'm'
+    * component[2].valueQuantity = 6.453E+25 's'
```

Comment:
Could you add some more components that test other types that could use exponents?
* `valueInteger` (integer)
* `valueSampledData.period` (decimal)
* `valueSampledData.dimensions` (positiveInt)